front/app/containers/Admin/pagesAndMenu/containers/NavigationSettings/VisibleNavbarItemList/index.tsx

```diff
@@ -29,11 +25,8 @@ export default function VisibleNavbarItemList() {
     }
 
     const pageData = navbarItem.relationships.static_page.data;
-
     pageData
-      ? previewNewCustomPages
-        ? clHistory.push(`${PAGES_MENU_PATH}/custom/${pageData.id}`)
-        : clHistory.push(`${PAGES_MENU_PATH}/pages/edit/${pageData.id}`)
+      ? clHistory.push(`${PAGES_MENU_PATH}/custom/${pageData.id}`)
```

Comment:
I guess so, I haven't thought about it. We could just use the old one since there will be no difference between custom and old pages now. Maybe something for the UX/UI review?

Reply:
I think old URLs are pretty fine, but we can definitely ask Ben to double-check.

Reply:
Yeah the old one is cleaner, I agree. Let's check with Ben to double check